src/features/asciidoctorConfig.ts

```diff
@@ -17,25 +18,30 @@ export async function getAsciidoctorConfigContent (documentUri: vscode.Uri): Pro
   if (workspaceFolder === undefined) {
     return undefined
   }
-  const configContentFromAsciidoctorConfig = await getConfigContent(workspaceFolder, '.asciidoctorconfig')
-  const configContentFromAsciidoctorConfigDotAdoc = await getConfigContent(workspaceFolder, '.asciidoctorconfig.adoc')
 
-  const configContents = [
-    configContentFromAsciidoctorConfig,
-    configContentFromAsciidoctorConfigDotAdoc,
-  ].filter((config) => config !== undefined)
+  const configContents: string[] = []
+  let currentFile: string = documentUri.fsPath
+  while (currentFile !== undefined && currentFile !== workspaceFolder.uri.fsPath) {
```

Comment:
I'm always a bit anxious with `while`. Maybe we could add a hard limit? Something like 100 iterations?